package.json

```diff
@@ -11,7 +11,7 @@
     "debug": "^2.2.0",
     "js-yaml": "^3.5.3",
     "jsonwebtoken": "^5.5.4",
-    "microgateway-core": "^2.0.15",
+    "microgateway-core": "^2.3.2-beta",
```

Comment:
In the next two weeks I'd imagine. I'm comfortable owning ensuring that a subsequent PR is made to remove the beta tag.